demos/dashboards-regulatory-screener/demo.js

```diff
@@ -0,0 +1,116 @@
+const loadingLabel = document.getElementById('loading-label');
+
+function displayRegulatoryScreener (postmanJSON) {
+    const secIds = [
+        'SecId',
+        'name',
+        'ticker',
+        'EETTemplateDate',
+        'EETGeneralDate'
+    ];
+
+    const headerFormats = {
+        SecId: 'Security ID',
+        name: 'Investment Name',
+        ticker: 'Ticker Symbol',
+        EETTemplateDate: 'EET Template Date',
+        EETGeneralDate: 'EET General Date'
+    };
```

Comment:
I think that list doesn't match the one posted here: https://developer.morningstar.com/direct-web-services/documentation/direct-web-services/screener/regulatory-screener - does it?